controllers/hvpa_controller.go

```diff
@@ -338,7 +339,17 @@ func (r *HvpaReconciler) reconcileHpa(hvpa *autoscalingv1alpha1.Hvpa) (*autoscal
 		return nil, err
 	}
 
-	updatePolicy := hvpa.Spec.Hpa.UpdatePolicy.UpdateMode
+	toDeploy := hvpa.Spec.Hpa.Deploy
+
+	upUpdatePolicy, downUpdatePolicy := autoscalingv1alpha1.UpdateModeAuto, autoscalingv1alpha1.UpdateModeAuto
```

Comment:
This is only for the time being, right? Until the `Scale` sub-resource has been implemented.